### PR TITLE
feat: add api and util to help builder 

### PR DESCRIPTION
**Pull request by irrun**
### Description

feat: add api and util to help builder 

### Rationale

tell us why we need these changes...

### Example

add an example CLI or API response...

### Changes

Notable changes: 
* add each change in a bullet point here
* ...

### CODE REVIEW

File: core/types/bid.go [hunk 1]
```diff
@@ -176,3 +177,9 @@ type BidIssue struct {
 	BidHash   common.Hash
 	Message   string
 }
+
+type MevParams struct {
+	SentryURL             string // The url of Mev sentry
```
Comment (zjubfd) [inline core/types/bid.go:182]: seems not necessary of `SentryURL`, it a validator is running behind a sentry, there is no way for a client access without knowing the sentry node in advance.

File: eth/api_backend.go [hunk 2]
```diff
@@ -457,6 +461,10 @@ func (b *EthAPIBackend) SendBid(ctx context.Context, bid *types.BidArgs) (common
 	return b.Miner().SendBid(ctx, bid)
 }
 
+func (b *EthAPIBackend) BestBidGasFee(parentHash common.Hash) *big.Int {
```
Comment (zjubfd) [inline eth/api_backend.go:464]: what about `BestBidRewardAfter(parentHash common.Hash)`

File: core/types/bid.go (hunk 1 above)
Comment (reneecok) [reply core/types/bid.go:182]: okay

File: eth/api_backend.go (hunk 2 above)
Comment (reneecok) [reply eth/api_backend.go:464]: okay

Comment (reneecok) [reply eth/api_backend.go:464]: seems ambiguous with bidutil, maybe parentHash param is enough to understand

**Review by zjubfd — APPROVED** [review]

**Pull request merged**